composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.0 || ^8.1",
```

Comment:
thats is not the way to do it https://getcomposer.org/doc/articles/composer-platform-dependencies.md